src/Microsoft.DotNet.Darc/src/Darc/Operations/GetSubscriptionsOperation.cs

```diff
@@ -35,19 +35,16 @@ public override async Task<int> ExecuteAsync()
 
                 var subscriptions = (await remote.GetSubscriptionsAsync()).Where(subscription =>
                 {
-                    return (string.IsNullOrEmpty(_options.TargetRepository) ||
-                        subscription.TargetRepository.Contains(_options.TargetRepository, StringComparison.OrdinalIgnoreCase)) &&
-                    (string.IsNullOrEmpty(_options.TargetBranch) ||
-                        subscription.TargetBranch.Contains(_options.TargetBranch, StringComparison.OrdinalIgnoreCase)) &&
-                    (string.IsNullOrEmpty(_options.SourceRepository) ||
-                        subscription.SourceRepository.Contains(_options.SourceRepository, StringComparison.OrdinalIgnoreCase)) &&
-                    (string.IsNullOrEmpty(_options.Channel) ||
-                        subscription.Channel.Name.Contains(_options.Channel, StringComparison.OrdinalIgnoreCase));
+                    return (_options.SubscriptionParameterMatches(_options.TargetRepository, subscription.TargetRepository) &&
```

Comment:
Nit: maybe try to align all `_options` in the same column

Reply:
I'll do that in a PR to master rather than the rollout to prod.